sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -433,3 +433,45 @@ def __str__(self):
         if self._error_format:
             return str(self._error_format)
         return super(ODataV4Error, self).__str__()
+
+class StreamConsumedError(Exception):
```

Comment:
Should it be a sub class of AzureError?

Reply:
And if the error type is provisional, should we host it in exceptions.py?

Reply:
@johanste bringing this question to you

Reply:
@xiangyan99 ^^